qt/app.py

```diff
@@ -373,10 +374,9 @@ def __init__(self, config, appInstance, qapp):
             self.config.setCurrentHashId(hash_id)
 
         if not config.canBackup(profile_id):
-            msg = _("Can't find snapshots folder.") + '\n' \
-                + _('If it is on a removable drive please plug it in and then '
-                    'press OK.')
-            messagebox.critical(self, msg)
+            messagebox.critical(self, _(
+                "Can't find snapshots folder.\nIf it is on a removable "
+                "drive please plug it in and then press OK."))
```

Comment:
Can you explain why you modified this?

Reply:
This either should be modified. Not sure why this happen. I did not try to make any changes here. Please do not mention these parts.